.github/workflows/e2e-tests.yml

```diff
@@ -80,6 +80,15 @@ jobs:
           STRIPE_SECRET_KEY: ${{ secrets.E2E_STRIPE_SECRET_KEY }}
         run: npm run test:e2e-setup
 
+      - name: Upload e2e setup logs
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ matrix.checkout }}-WP_latest-WC_latest-setup-logs
+          path: tests/e2e/e2e-setup.log
+          if-no-files-found: ignore
+          retention-days: 14
```

Comment:
I would reduce the retention to ~4 days to avoid consuming unnecessary resources.

Reply:
I think that's a good idea, but I've only dropped it to 7 days so we have time to investigate. Also, this will only upload a file when the setup steps/script fail, and that file is plaintext only. So I don't think it's a huge additional thing to include.